selenium_test/test/locators.py

```diff
@@ -77,7 +77,7 @@ class SubmissionPageLocators(object):
     INSPECTION_LINKS = (By.XPATH, "//table//a[contains(@href, '/inspect/')]")
 
 class StudentFeedbackPageLocators(object):
-    ASSISTANT_FEEDBACK_LABEL = (By.XPATH, "//h4[text()='Assistant feedback']")
```

Comment:
Do you want this to be a separate commit, or rebased to one of the earlier?

Reply:
The idea is to be rebased to one of the earlier, that's why it's a fixup commit :)